app/konnect/dev-portal/applications/dynamic-client-registration/overview.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Dynamic Client Registration Overview
+content-type: explanation
+---
+
+Dynamic Client Registration (DCR) within {{site.konnect_short_name}} Dev Portal allows applications created in the portal to automatically create a linked application in a third-party Identity Provider (IdP).
+This outsources the issuer and management of application credentials to a third party, allowing for additional configuration options and compatibility with various OIDC features provided by the IdP.
+
+## Authentication Methods
+
+DCR support in {{site.konnect_short_name}} provides multiple methods by which applications can be authenticated using industry-standard protocols. These methods include:
+* **Client Credentials Grant**: Authenticate with the client ID and secret provided to the application.
+* **Bearer tokens**: Authenticate using a token requested from the IdP's `/token` endpoint.
+* **Session cookie**: Allow sessions from either client credentials or bearer tokens to persist via cookie until an expiration.
```

Comment:
Is the expiration configured by the customer? Or is it a default set by the cookie/Konnect?